packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig

```diff
@@ -4,10 +4,12 @@
 {% block h1 %}{{ 'Bestsellers'|trans }}{% endblock %}
 
 {% block main_content %}
+    {% import _self as self %}
```

Comment:
does this kind of change belong into this commit?

Reply:
No it does not :(